### DISCUSSION TITLE
book/javascript/null-undefined/

### DISCUSSION THREAD

**Discussion by giscus** [xsltdev/scriptdev.ru#138] [Announcements]
# book/javascript/null-undefined/

Справочник Typescript с примерами кода

https://scriptdev.ru/book/javascript/null-undefined/

**Comment by andrew-klepcha** [comment]
в документе по ссылке "TypeScript coding guidelines" сказано: These are Coding Guidelines for Contributors to TypeScript. This is NOT a prescriptive guideline for the TypeScript community. Так что пока вы не вносите вклад собственно в сам TypeScript -- пользуйтесь null на здоровье
